Server/mods/deathmatch/logic/luadefs/CLuaPlayerDefs.cpp

```diff
@@ -946,47 +946,24 @@ int CLuaPlayerDefs::TakePlayerScreenShot(lua_State* luaVM)
     return 1;
 }
 
-int CLuaPlayerDefs::SetPlayerScriptDebugLevel(lua_State* luaVM)
+bool CLuaPlayerDefs::SetPlayerScriptDebugLevel(CElement* const element, const std::variant<unsigned int, bool> variant)
 {
-    CElement*    pElement;
-    unsigned int uiMode;
-    bool         bHideDebugger = false;
-
-    CScriptArgReader argStream(luaVM);
-    argStream.ReadUserData(pElement);
-    if (argStream.NextIsBool())
+    // cant be a true bool
+    unsigned int level;
+    if (std::holds_alternative<bool>(variant))
     {
-        bool bTemp;
-        argStream.ReadBool(bTemp);
-
-        if (bTemp)
-            argStream.SetCustomError("You can only pass false to hide the debug window or a level (0-3)");
+        if (std::get<bool>(variant))
+            throw std::invalid_argument("You can only pass false to turn the debugger off, or a level(0 - 3)");
```

Comment:
SetPlayerScriptDebugLevel

```suggestion
            throw std::invalid_argument("You can only pass false to hide the debug window or a level (0-3)");
```

Tbh this false thing is weird - should get rid of the false thing in a separate PR.